x/supplier/keeper/query_claim.go

```diff
@@ -0,0 +1,58 @@
+package keeper
+
+import (
+	"context"
+
+	"github.com/cosmos/cosmos-sdk/store/prefix"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/query"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+
+	"github.com/pokt-network/poktroll/x/supplier/types"
+)
+
+func (k Keeper) ClaimAll(goCtx context.Context, req *types.QueryAllClaimRequest) (*types.QueryAllClaimResponse, error) {
```

Comment:
:thinking: I wouldn't say anything about his one were "claim" not a verb; however, such as it is, I feel like this results in ambiguity around the meaning. Is it a bad idea to take this naming into our own hands in such cases, would we be breaking anything other than convention?

Reply:
Yeah ignite default naming is a bit odd, I would change to even just `AllClaims` maybe? But it may be better to leave the scaffollded functions named as they are idk the convention here.